server/middlewares/group.js

```diff
@@ -20,7 +20,8 @@ function isAdminOrCurrentUser(req, res, next) {
 
 function isGroupMember(req, res, next) {
```

Comment:
Can you change the name here to reflect the fact that the user is allowed to be a `sysAdmin` or a group member? Preferably something like `isGroupMemberOrSysAdmin`